roles/mirror_images/tasks/mirror-images.yml

```diff
@@ -2,8 +2,11 @@
 - name: "Mirror image"
   vars:
     _image_tag: "{{ image | regex_search('(?<=sha256)(.{12})') | default('', True) }}"
-    _image_name: "{{ '/'.join(image.split('@')[0].split('/')[1:]) }}"
-    _tgt_image_location: "{{ mi_registry }}/{{ _image_name }}{%- if _image_tag | length %}{{ _image_tag }}{% endif %}"
+    _image_name: "{{ '/'.join(image.split('@')[0].split('/')[-1:]) }}"
+    _image_org: "{{ '/'.join(image.split('@')[0].split('/')[1:-1]) }}"
+    _image_org_path: "{{ (mi_dst_org | default('') | length) | ternary(mi_dst_org, _image_org) }}"
```

Comment:
I think is better to set `mi_dst_org` in defaults/main.yml because a ternary requires a value for it. I'm not sure if you have tested yet, but I believe it should fail when not defined.

Reply:
Done, thanks